.env.example

```diff
@@ -12,3 +12,4 @@ MAIL_SENDER_NAME=
 MAIL_PASSWORD=
 SEC_ENCRYPT_KEY=
 SOURCE_MATERIALS_API=https://api-source-materials.sws2apps.com/api
+USER_PARSER_API_KEY=
```

Comment:
@ild0tt0re: we no longer use this `USER_PARSER_API_KEY` variable. Could you please remove it? Thank you.

Reply:
Ok, can i remove user parser setup also from local development setup guide?

Reply:
@ild0tt0re: I’ve already removed it on the guide. We will need to remove it from here. Thanks.